about.html

```diff
@@ -0,0 +1,119 @@
+                        networking, opportunities, recruiters and more. This is set to be an unmissable event for anyone
+                        in Tech.
+                    </div>
+                    <div class="hero_location about">
+                        <div class="hero_date about">2023.01.11(Wed) ~ 12(Thursday)</div>
+                        <div class="hero_venue about">@San Francisco, CA & Virtual</div>
+                    </div>
+                </div>
+            </section>
+        </div>
+        <section class="aboutEvent">
+            <div class="event_wrapper">
+                <h2>About the Event</h2>
+                <div class="guide_bar"></div>
+                <p class="event_description">Digital transformation has touched every sphere of life. We are all
+                    immersed in a world of data. This data facilitates our healthcare, business and government
+                    activities, commerce and entertainment. The Techno TechoThirst Summit: Looking Forward highlights
+                    the ways in which society has been changed by digital disruption and provides the roadmap for what
+                    lies ahead.
+
+                    The biggest stories of the day – cryptocurrency, the Metaverse, supply chain management and
+                    cybersecurity – are all covered at this comprehensive and timely event. We will also learn about new
+                    developments in biotechnology, space travel, gaming, and work from home solutions.
+
+                    Through all of these conversations, we explore the new possibilities engendered by technology and
+                    the shape, look and feel of the future. We also touch upon the possible downside of unfettered
+                    growth of technology such as the increased vulnerability to cyberattacks, the degradation of
+                    privacy, and the inherent presence of biases that negatively impact marginalized groups in society.
+                </p>
+            </div>
+        </section>
+        <section class="secSectionAbout">
+            <article class="past_events">
+                <h2>See the past Nova Events</h2>
+                <div class="guide_bar"></div>
+                <p>Take a look at the past Nova events which took place in Dubai and in Erbil</p>
+            </article>
+            <section class="imgTextabt">
+                <div class="eventPhotoswText">
+                    <img id="imgevent" src="statics/images/about_2011.png" alt="eventPhoto">
+                </div>
+                <div class="eventPhotoswText">
+                    <img id="imgevent" src="statics/images/about_2013.png" alt="eventPhoto">
+                </div>
+            </section>
+            <article class="partners">
+                <h2 class="partnerText">Partners</h2>
+                <div class="guide_bar"></div>
+                <div class="partnerLogos">
```

Comment:
- [ ] please kindly ensure that the about page footer has a dark background color. please fix that
- [ ] also, it does not have the partners section
- [ ] kindly add the space between your images and your footer in the about page
yours 
![Screenshot (733)_LI](https://user-images.githubusercontent.com/65298048/211081617-8f871f8d-2e51-4443-b973-48377c0fa3f8.jpg)

original design
![Screenshot (731)](https://user-images.githubusercontent.com/65298048/211081717-10e6ea7f-e20d-4cb3-b6dc-385b734b08aa.png)

Reply:
changed . Thanks.

<img width="941" alt="image" src="https://user-images.githubusercontent.com/39811497/211134624-70fc01b5-e527-46fc-896a-546430d8f3f8.png">